AWSPinpoint/AWSPinpointEndpointProfile.h

```diff
@@ -276,7 +276,35 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong) NSString *_Nullable userId;
 
+/**
```

Comment:
Instead of exposing all of these functions (which do error checking, and limit mutability, etc), I think it would be easier to just expose a single dictionary that customers can update:
```
@property (nonatomic, strong) NSDictionary<NSString*, NSArray*> *userAttributes;
```